build.gradle

```diff
@@ -0,0 +1,28 @@
+plugins {
+    id 'java'
+}
+
+group = 'com.example'
+version = '1.0.0'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    // Core dependencies
+    implementation 'org.seleniumhq.selenium:selenium-java:4.12.0'
+    testImplementation 'junit:junit:4.13.2'
+}
+
+tasks.withType(JavaCompile) {
+    options.encoding = 'UTF-8'
+}
+
+jar {
+    manifest {
+        attributes(
+                'Main-Class': 'com.example.DS-and-Algorithm' // Replace with your main class
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider valid Java class name format</b></div>
<div id="fix">

The main class name `DS-and-Algorithm` contains a hyphen which is not a valid Java class name character. Consider using camelCase or underscore naming convention.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -25 +25 @@
 -                'Main-Class': 'com.example.DS-and-Algorithm' // Replace with your main class
 +                'Main-Class': 'com.example.DSAndAlgorithm' // Replace with your main class
```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/Amruta101998/DS-and-Algorithm/pull/24#issuecomment-2514645264>#10829c</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged